File: sample/ruby/main_make_model_sample.rb
require_relative 'build_box'
require_relative 'ply_util'

room_name = '1000'
build_box = BuildBox.new(room_name)

build_box.set_box_size(0.5)
build_box.set_build_interval(0.01)

ply_file_name = 'piyo.ply'

boxes = get_boxes_from_ply(ply_file_name)
# puts boxes

boxes.each do |box|
  build_box.create_box(*box)
end

build_box.send_data

File: sample/javaScript/mainMakeModelSample.mjs
import BuildBox from './buildBox.mjs';
import { getBoxesFromPly } from './plyUtil.mjs';

(async () => {
  const roomName = '1000';
  const buildBox = new BuildBox(roomName);

  buildBox.setBoxSize(0.5);
  buildBox.setBuildInterval(0.01);

  const plyFileName = 'piyo.ply'

  const boxes = getBoxesFromPly(plyFileName)

  for (const box of boxes) {
    // console.log(b)
    buildBox.createBox(...box)
  }

  await buildBox.sendData();
  console.log('send data done')
})().catch(error => {
  console.error(error);
});
